lib/ecl/ecl_file.c

```diff
@@ -992,3 +992,75 @@ static ecl_file_type * ecl_file_iopen_rstblock__( const char * filename , int se
 ecl_file_type * ecl_file_iopen_rstblock( const char * filename , int seqnum_index , int flags) {
   return ecl_file_iopen_rstblock__(filename , seqnum_index , flags );
 }
+
+void  ecl_file_write_index( ecl_file_type * ecl_file , const char * filename , const char * index_filename) {
```

Comment:
This meat of this method should be in a function `ecl_file_view_write_index( const ecl_file_view_type * file_view, FILE * ostream)`.